editoast/src/models/timetable.rs

```diff
@@ -58,6 +58,20 @@ impl Timetable {
             .map_err(Into::into)
     }
 
+    pub async fn trains_from_timetable_id(timetable_id: i64, conn: &mut DbConnection) -> Result<Vec<TrainSchedule>> {
+        use editoast_models::tables::train_schedule::dsl;
+        use diesel::prelude::*;
+        use diesel_async::RunQueryDsl;
+
+
+        dsl::train_schedule.filter(dsl::timetable_id.eq(timetable_id))
+        .load_stream::<Row<TrainSchedule>>(conn.write().await.deref_mut())
+            .await?
+            .map_ok(|ts| ts.into())
+            .try_collect::<Vec<TrainSchedule>>()
+            .await.map_err(Into::into)
+    }
+
```

Comment:
I can't see where this function is used in this PR. If it's used, you can use `models::prelude::List` for that, or better delete it if it's unused ^^

Reply:
It was a first try that was not conclusive ^^